x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/SnapshotActionTests.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.ilm;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.common.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.util.List;
+
+public class SnapshotActionTests extends AbstractActionTestCase<SnapshotAction> {
```

Comment:
Can you override the `mutateInstance` method in this class also so `equals` and `hashCode` are also tested?

Reply:
Done